### PR TITLE
Added confirm to delete

### PR DESCRIPTION
**Pull request by itguy51**
Re: #1697 

I've added a dialog that appears as attached when selecting "Remove Game" from the right-click context menu.

confirm

### CODE REVIEW

**Comment by itguy51** [comment]
It appears that the Travis builds on Linux errored out pre-build (It looks like something in the setup of the worker crashed.)

**Comment by tambry** [comment]
@itguy51 I've restarted the failed builds.

**Comment by itguy51** [comment]
@tambry Awesome.

File: rpcs3/Gui/GameViewer.cpp [hunk 1]
```diff
@@ -233,8 +233,10 @@ void GameViewer::RemoveGame(wxCommandEvent& event)
 {
 	long i = GetFirstSelected();
 	if (i < 0) return;
-	
-	fs::remove_all(vfs::get(m_path) + this->GetItemText(i, 6).ToStdString());
+
+	wxMessageDialog *dial = new wxMessageDialog(NULL, wxT("Permanently delete game files?"), wxT("Confirm Delete"), wxYES_NO | wxNO_DEFAULT);
```
Comment (11DH11) [inline rpcs3/Gui/GameViewer.cpp]: Memory leak would be here. Simply do not create dialogs in heap.

File: rpcs3/Gui/GameViewer.cpp [hunk 2]
```diff
@@ -233,8 +233,10 @@ void GameViewer::RemoveGame(wxCommandEvent& event)
...
+
+	if(dial->ShowModal() == wxID_YES) fs::remove_all(vfs::get(m_path) + this->GetItemText(i, 6).ToStdString());
```
Comment (11DH11) [inline rpcs3/Gui/GameViewer.cpp]: Place body to the next line

File: rpcs3/Gui/GameViewer.cpp (hunk 1 above)
Comment (tambry) [inline rpcs3/Gui/GameViewer.cpp]: I recently started using wxWidgets and from what I understand, whenever an object is closed (ie. a message dialog being closed), then even if you used new to create it, it automatically gets freed by wxWidgets.

Comment (11DH11) [inline rpcs3/Gui/GameViewer.cpp]: That correct for frames, not dialogs.
Look at examples, e.g. http://docs.wxwidgets.org/trunk/overview_cmndlg.html

Comment (11DH11) [inline rpcs3/Gui/GameViewer.cpp]: Dialog object can be used after closing

Comment (Nekotekina) [inline rpcs3/Gui/GameViewer.cpp]: This looks like memory leak. Also we don't use `wxT`.

Comment (tambry) [inline rpcs3/Gui/GameViewer.cpp]: Might also want to use wxMessageBox instead of the dialog.

**Comment by itguy51** [comment]
New commit message should be that I replaced wxMessageDialog, not wxDialogBox.

**Pull request merged**